app/src/main/java/org/openhdp/hdt/ui/tracking/TrackingViewModel.kt

```diff
@@ -100,8 +109,9 @@ class TrackingViewModel @ViewModelInject constructor(
                 id,
                 name,
                 category.name,
-                totalTime,
+                millisAfterReset,
```

Comment:
Something doen't work with setting time after reset, but I don't know where (in code) is the problem.